axes/decorators.py

```diff
@@ -276,8 +278,13 @@ def watch_login(func):
 
     def decorated_login(request, *args, **kwargs):
         # share some useful information
-        if func.__name__ != 'decorated_login' and VERBOSE:
-            log.info('AXES: Calling decorated function: %s' % func.__name__)
+        try:
+            func_name = func.__name__
+        except AttributeError as e:
+            if isinstance(func, partial):
+                func_name = func.func.__name__
+        if func_name != 'decorated_login' and VERBOSE:
```

Comment:
We could have undefined func_name here if `func.__name__` is not set func is not instance of partial. I think we can set func_name as None by default.

Also could you reorder the order here, so we check `VERBOSE` first?

Reply:
@kviktor :+1: done